### DISCUSSION TITLE
Elgato Stream Deck

### DISCUSSION THREAD

**Discussion by maxxinessss** [catppuccin/catppuccin#2394] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Just simple catppuccin themed icons, only needs a handful of essentials

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
I think this might be better raised as an issue on [catppuccin/steam-deck](https://github.com/catppuccin/steam-deck)

**Reply by maxxinessss** [reply]
Like an elgato stream deck

**Reply by sgoudham** [reply]
Oh haha, I misread the title of this issue. Sorry about that.
For the elgato stream deck, looks like this website exists to allow the icons to be edited easily: https://www.elgato.com/uk/en/s/keycreator
